content/en/docs/reference/command-line-tools-reference/feature-gates/admission-webhook-match-conditions.md

```diff
@@ -12,7 +12,11 @@ stages:
     toVersion: "1.27"
   - stage: beta
     defaultValue: true
-    fromVersion: "1.28"  
+    fromVersion: "1.28"
+    toVersion: "1.29"
+  - stage: stable
+    defaultValue: true
+    fromVersion: "1.30"
 ---
 Enable [match conditions](/docs/reference/access-authn-authz/extensible-admission-controllers/#matching-requests-matchconditions)
```

Comment:
We should also ensure alignment of the feature state in this reference page ([here](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#matching-requests-matchconditions)) with these changes. Please consider modifying [Line 724](https://github.com/ivelichkovich/website/blob/dev-1.30/content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md?plain=1#L724) as per the suggestion below.

```diff
- {{< feature-state state="beta" for_k8s_version="v1.28" >}}
+ {{< feature-state feature_gate_name="AdmissionWebhookMatchConditions" >}}
```

Reply:
done, thanks!